### ISSUE TITLE
Error with 'About Rocket.chat' 

### ISSUE THREAD

**Issue by RichardFoxworthy** [RocketChat/Rocket.Chat.Electron#437]
## My Setup

 - Mac OSX 10.12.4
 - Rocket.chat+ 2.7

- [x] I have tested with the latest application version
- [x] I can simulate the issue easily

## Description
Error with 'About Rocket.chat' 

#### Current Behavior

Downloaded Mac executable for Rocket.chat+ - it seems to work fine, but throws this error if I click the 'About Rocket.chat+' menu item

```
Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined
    at Event.set (/Applications/Rocket.Chat+.app/Contents/Resources/electron.asar/browser/api/web-contents.js:252:22)
    at EventEmitter.electron.ipcMain.on (/Applications/Rocket.Chat+.app/Contents/Resources/app.asar/app/src/background/autoUpdate.js:124:26)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at WebContents.<anonymous> (/Applications/Rocket.Chat+.app/Contents/Resources/electron.asar/browser/api/web-contents.js:256:37)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)
```

#### Expected Behavior

View version information about Rocket.chat+

**Comment by playmono** [comment]
I'm seeing this instead (in the popup, as raw-text and non-executed code):
`nction(n,t,r){return r===F&&(r=t,t=F),r!==F&&(r=Iu(r),r=r===r?r:0),t!==F&&(t=Iu(t),t=t===t?t:0),gt(Iu(n),t,r)},On.clone=function(n){return dt(n,4)},On.cloneDeep=function(n){return dt(n,5)},On.cloneDeepWith=function(n,t){return t=typeof t=="function"?t:F,dt(n,5,t)},On.cloneWith=function(n,t){return t=typeof t=="function"?t:F,dt(n,4,t)}, On.conformsTo=function(n,t){return null==t||bt(n,t,Lu(t))},On.deburr=Tu,On.defaultTo=function(n,t){return null==n||n!==n?t:n},On.divide=rc,On.endsWith=function(n,t,r){n=zu(n),t=jr(t);var e=n.length,e=r=r===F?e:gt(Ou(r),0,e);return r-=t.length,0<=r&&n.slice(r,e)==t},On.eq=hu,On.escape=function(n){return(n=zu(n))&&Y.test(n)?n.replace(H,et):n},On.escapeRegExp=function(n){return(n=zu(n))&&fn.test(n)?n.replace(on,"\\$&"):n},On.every=function(n,t,r){var e=af(n)?o:wt;return r&&ze(n,`

Debian testing, rocket chat+ 2.7

**Comment by Kingbalou** [comment]
Same error on Windows 10

**Comment by Rumbles** [comment]
Same issue on the rpm version :)

**Comment by jszaszvari** [comment]
Same here with OS X 10.12

**Comment by xenithorb** [comment]
Confirmed on Fedora with my own package and directly from git `develop` branch

**Comment by monspid** [comment]
Same error on Windows 7

**Comment by compuguy** [comment]
I can confirm this as well on Mac OSX: 
`Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined
    at Event.set (/Applications/Rocket.Chat+.app/Contents/Resources/electron.asar/browser/api/web-contents.js:252:22)
    at EventEmitter.electron.ipcMain.on (/Applications/Rocket.Chat+.app/Contents/Resources/app.asar/app/src/background/autoUpdate.js:124:26)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at WebContents.<anonymous> (/Applications/Rocket.Chat+.app/Contents/Resources/electron.asar/browser/api/web-contents.js:256:37)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)`

**Comment by compuguy** [comment]
This may be a duplicate of #427 (With #431 being another duplicate of the former).

**Comment by engelgabriel** [comment]
Closed in favor of  #427
